corehq/apps/reports/static/reports/js/data_corrections.js

```diff
@@ -101,9 +101,17 @@ hqDefine("reports/js/data_corrections", [
 
     // Controls the full modal UI
     var DataCorrectionsModel = function (options) {
-        assertProperties.assert(options, ['saveUrl', 'properties'],
-            ['propertyNames', 'propertyNamesUrl', 'displayProperties', 'propertyPrefix', 'propertySuffix', 'analyticsDescriptor']);
+        assertProperties.assert(options, ['saveUrl', 'properties'],[
+            'propertyNames',
+            'propertyNamesUrl',
+            'displayProperties',
+            'propertyPrefix',
+            'propertySuffix',
+            'analyticsDescriptor',
+            '$modal'
```

Comment:
Missed this earlier - shouldn't `$modal` be a required property not an optional one?

Reply:
fixed